data/db_demo.yaml

```diff
@@ -57,6 +57,12 @@ streams/=ztf_public/users:
   - user_id: =groupadmin
   - user_id: =testadmin
 
+streams/=ztf_partnership/users:
+  - user_id: =viewonlyuser
+  - user_id: =fulluser
+  - user_id: =groupadmin
+  - user_id: =testadmin
+
```

Comment:
Needed to grant users this stream so that they could be added to the group that required the ZTF partnership stream.